src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,78 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { getTodos } from '../../api';
+
+type Props = {
+  setTodos: React.Dispatch<React.SetStateAction<Todo[] | null>>;
+};
+
+export const TodoFilter: React.FC<Props> = ({ setTodos }) => {
+  const [qwerty, setQwerty] = useState('');
+  const [status, setStatus] = useState('');
+
+  useEffect(() => {
+    getTodos().then(serverTodos => {
+      let curentTodos: Todo[] | null = serverTodos.filter(todo =>
```

Comment:
There is a typo in the variable name `curentTodos`. It should be `currentTodos` to correctly spell 'current'.